ccdproc/ccddata.py

```diff
@@ -98,21 +108,21 @@ class CCDData(NDDataArray):
     This is useful, for example, when plotting a 2D image using
     matplotlib.
 
-        >>> from ccdproc import CCDData   
+        >>> from ccdproc import CCDData
         >>> from matplotlib import pyplot as plt   # doctest: +SKIP
-        >>> x = CCDData([[1,2,3], [4,5,6]], unit='adu') 
+        >>> x = CCDData([[1,2,3], [4,5,6]], unit='adu')
         >>> plt.imshow(x)   # doctest: +SKIP
 
     """
     def __init__(self, *args, **kwd):
         if 'meta' not in kwd:
             kwd['meta'] = kwd.pop('header', None)
         if 'header' in kwd:
-            raise ValueError("Can't have both header and meta")
+            raise ValueError("Can't have both header and meta.")
```

Comment:
Since it's a nit picking PR, I would raise a super tiny point here. Start the error messages lowercase. 

This was pointed out somewhere else recently (but now I can't find it), that if you look at what comes out of the interpreter, it may look more consistent to start the message lowercase as it follows a colon. 
However this is something that's not done consistently yet in the astropy world, but would be nice to do.

```
In [1]: raise ValueError("Can't have both header and meta")
---------------------------------------------------------------------------
ValueError                                Traceback (most recent call last)
<ipython-input-1-f52884564396> in <module>()
----> 1 raise ValueError("Can't have both header and meta")

ValueError: Can't have both header and meta
```